.github/workflows/ci.yml

```diff
@@ -14,6 +14,22 @@ jobs:
   # Note that vcpkg dependencies takes the majority of the build time.
   # We cache them using GitHub Actions cache and export, making the scripts below a bit more complex.
 
+  # Format file with instruction in ~/.editorconfig file
+  check-format:
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{ env.dotnet_3_version }}
+      - name: Restore tools
+        run: dotnet tool restore
+      - name: Check format
+        run: dotnet format "." -f --check
```

Comment:
I think we should not use the `-f` option, but instead directly the project files/dirs. If we add a solution file referencing our 3 projects, then `dotnet format --check` will work as-is.

@GPSnoopy what are your thoughts on adding a solution file?

Reply:
Indeed this works fine for the CI, but for contributors this will start formatting 3rd party dependencies found under the `build` directory. Adding a solution file would solve that and make the command easier. I suggest you go ahead and add a solution file, unless @GPSnoopy has a strong opinion against.

Reply:
It makes sense for me, pushed new version, check it out @GPSnoopy t @jgiannuzzi  👀

Reply:
It looks like the new version somehow didn't get pushed 😅

Reply:
hey @jgiannuzzi, version is noted in .config\dotnet-tools.json 
since we are doing `dotnet tool restore` it's picking version set there

Reply:
I meant the new version of your PR adding the solution file and changing the command to check the format 😄

Reply:
As per @GPSnoopy's [comment](https://github.com/G-Research/ParquetSharp/pull/196#discussion_r636976230) below, he is not keen on a solution file since it's currently auto-generated by CMake.
Do we want to go for something like `for project in */*.csproj; do dotnet format --check $project; done` then?

Reply:
also, we can do with multiple commands, since it will be cleaner (readable) and will not affect the time of the job itself much

`      - name: Check format - ParquetSharp
        run: dotnet format "./csharp/ParquetSharp.csproj"
      - name: Check format - Benchmark
        run: dotnet format "./csharp/ParquetSharp.Benchmark.csproj"
      - name: Check format - Test
        run: dotnet format "./csharp/ParquetSharp.Test.csproj"
`

Reply:
works for me 👍 — just make sure to add `--check` 🤓

Reply:
added all paths and `--check` 🤓

Reply:
i'll resolve this comments, as new change is using `jb cleanupcode` command so we can discuss that separatly